### DISCUSSION TITLE
styles.css (404: Not Found)

### DISCUSSION THREAD

**Discussion by ManuelSpinola** [quarto-dev/quarto-cli#4143] [Q&A]
I have a blog made in quarto and I didn't have a problem until today. When render the website I got the following error:

GET: /posts/2022-07-01-text-analysis/text-analysis.html
  /posts/2022-07-01-text-analysis/styles.css (404: Not Found)

I have the latest quarto version.

**Comment by baptiste** [comment]
It's unlikely anyone can help you without more information; can you perhaps share a minimal reproducible example illustrating the problem, as a github repository or similar? Also, by latest quarto do you mean latest release or built from source (which version)?

**Comment by ManuelSpinola** [comment]
Thank you Baptiste.

It was a problem in my iCloud.  The issue was solved.
